src/isFunction.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Checks if `input` is a function
+ *
+ * @param {*} input
+ * @return {bool} Whether the input is a function
+ */
+export default function isFunction(input) {
+  return typeof input === 'function';
+}
```

Comment:
It seems a little unnecessary to have this in a separate module/file. I think you should just inline it where it is needed.